applications/tube_network/README.md

```diff
@@ -0,0 +1,107 @@
+- To continue:
+    - Run queries on the London Tube Network using [Graql Console](http://dev.grakn.ai/docs/running-grakn/console) and [Workbase](http://dev.grakn.ai/docs/workbase/overview).
+    - or:
+        - Perform statistical queries: `python3 -m tube_network.src.statistics`. Learn more about the [Compute Queries](http://dev.grakn.ai/docs/query/compute-query).
+        - Try the journey planner: `python3 -m tube_network.src.journey_planner`
+        - Interact with the journey planner interface: `python3 -m tube_network.src.app`
+
+## Downloading Data
+The data necessary to build a Grakn of the Tube Network is already included in this repository, the code to acquire it can be found in [`data/download.py`](data/download.py).
+
+## Importing Data
+We can import this data into the Grakn keyspace we have just created. The name of the keyspace is set in `settings.py`, so you can change it there if you need to. You don't have to implement settings in this way in your own application.
+Check Grakn is up and running: `./grakn server status`
+To import, run the [`src/migration.py`](src/migration.py), either in your IDE, or from the grakn_examples directory as follows:
+
+```bash
+cd path-to-cloned-repository/applications/tube_network
+python3 -m tube_network.src.migration
+```
+
+The content of [`src/migration.py`](src/migration.py) is a python script that:
+1. as it goes through the TFL's `.json` files, constructs dictionaries with a pre-defined structure that get passed on to the template functions for constructing Graql relationship/entity insert queries.
+2. the constructed Graql insert queries, after basic uniqueness validation, get stored as items of arrays within a dictionary.
+3. the dictionary containing all the Graql queries, gets flattened, to prepare the data in two chunks of entities and relationships for a series of concurrent insertions.
+4. lastly, a set of processes initiate the set of transactions that perform the Graql insert queries on the `tube_network` keyspace.
+
+Once complete, you have stored the tube network data in Grakn!
+Now you're ready to start playing with the data.
+
+
+## Reasoning
+Now you can query the database, you can try asking more complex questions, like asking for the time between stations of over 8 minutes:
+```
+match
+    $sta1 isa station, has name $sta1-nam;
+    $sta2 isa station, has name $sta2-nam;
+    $sec isa route-section, has duration $dur; $dur > 8;
+    (beginning: $sta1, end: $sta2, service: $sex) isa tunnel;
+    (route-operator: $tul, section: $sec) isa route;
+    $tul isa tube-line, has name $tul-nam;
+get $sta1-nam, $sta2-nam, $tul-nam, $dur; limit 30;
+```
+
+## Retrieve Statistical Information
+To gain an overall understanding of the available statistical queries in Grakn, try the [`src/statistics.py`](src/statistics.py).
```

Comment:
The UI for the statistics looks much more mature now, perhaps some docs on using it, or a screenshot of some cool terminal interaction would be good